anchore_manager/cli/analyzers.py

```diff
@@ -137,7 +137,7 @@ def exec(docker_archive, anchore_archive, digest, parent_digest, image_id, tag,
                 try:
                     if int(created_at) < 0 or int(created_at) > now+1:
                         raise Exception()
```

Comment:
Shouldn't this be a ValueError directly? I think the outer wrapper can be removed as it just obscures possible other issues.

Reply:
I think it should, but I am being overly cautious here :) The thing that made me pause is that if the outer try/except is removed, it wouldn't catch a `ValueError` if `int(created_at)` fails. 

If you think there might not be a chance for that to happen, I am OK with further cleaning up this

Reply:
yes, the exception is the same either way, just a question of clear message to the user. Ideally, it would convert to an int once, handle that error case, then do the range check with a different msg, but as you have it is ok since its only better than what it was. I think we can cleanup the input checks here a lot for testability & maintainability, but that's another topic.